src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface.jl

```diff
@@ -46,16 +46,14 @@ end
 function SplitExplicitFreeSurface(grid; gravitational_acceleration = g_Earth,
                                         settings = SplitExplicitSettings(200))
 
-    η = ZFaceField(grid, indices = (:, :, size(grid, 3)+1))
+    η = FreeSurfaceDisplacementField(velocities, free_surface, grid)
```

Comment:
```suggestion
η = ZFaceField(grid, indices = (:, :, size(grid, 3)+1))
```
velocities is not defined here